src/windows_c.zig

```diff
@@ -793,7 +793,7 @@ pub const SystemErrno = enum(u16) {
                 Win32Error.DIR_NOT_EMPTY => SystemErrno.ENOTEMPTY,
                 Win32Error.WSAENOTSOCK => SystemErrno.ENOTSOCK,
                 Win32Error.NOT_SUPPORTED => SystemErrno.ENOTSUP,
-                Win32Error.BROKEN_PIPE => SystemErrno.EOF,
+                Win32Error.BROKEN_PIPE => SystemErrno.EPIPE,
```

Comment:
are you sure

Reply:
https://man7.org/linux/man-pages/man3/errno.3.html

<img width="411" alt="image" src="https://github.com/oven-sh/bun/assets/24465214/9ec6f2e9-3084-4240-a48b-1dec157d3177">